lib/stripe/api_resource.rb

```diff
@@ -48,6 +48,15 @@ def self.save_nested_resource(name)
       end
     end
 
+    def self.custom_method(name, http_method, path = nil)
```

Comment:
Okay, this is admittedly a bit of a bike shed, but when I'm reading the invocations above, it's obvious that the first parameter is the method name, but not so obvious (until reading a lot of them) that the second is an HTTP verb and the third is an API route override.

What do you think about giving the parameters after the first a name? Like:

``` ruby
custom_method :capture, http_verb: :post

custom_method :capture, http_verb: :post, http_path: 'capture'
```

Or something like that?